python/cac_tripplanner/templates/manifest.json

```diff
@@ -0,0 +1,81 @@
+{
+  "short_name": "GoPhillyGo",
+  "name": "GoPhillyGo Trip Planner",
+  "icons": [
+    {
+      "src": "/static/images/apple-touch-icon-57x57.png",
+      "type": "image/png",
+      "sizes": "57x57"
+    },
+    {
+      "src": "/static/images/apple-touch-icon-60x60.png",
+      "type": "image/png",
+      "sizes": "60x60"
+    },
+    {
+      "src": "/static/images/apple-touch-icon-57x57.png",
+      "type": "image/png",
+      "sizes": "57x57"
+    },
+    {
+      "src": "/static/images/apple-touch-icon-72x72.png",
+      "type": "image/png",
+      "sizes": "72x72"
+    },
+    {
+      "src": "/static/images/apple-touch-icon-76x76.png",
+      "type": "image/png",
+      "sizes": "76x76"
+    },
+    {
+      "src": "/static/images/apple-touch-icon-114x114.png",
+      "type": "image/png",
+      "sizes": "114x114"
+    },
+    {
+      "src": "/static/images/apple-touch-icon-120x120.png",
+      "type": "image/png",
+      "sizes": "120x120"
+    },
+    {
+      "src": "/static/images/apple-touch-icon-144x144.png",
```

Comment:
I think you can remove all the icon entries from the manifest except this one (144x144). I'll add one or two more (192, maybe 96) in a follow-up PR.